OloEngine/src/OloEngine/Physics3D/JoltBody.cpp

```
@@ -253,14 +254,21 @@ namespace OloEngine
 
         const auto& bodyLockInterface = GetBodyLockInterface();
-        JPH::BodyLockRead lock(bodyLockInterface, m_BodyID);
-        if (lock.Succeeded())
+        if (JPH::BodyLockRead lock(bodyLockInterface, m_BodyID); lock.Succeeded())
         {
             const JPH::Body& body = lock.GetBody();
             const JPH::MotionProperties* motionProperties = body.GetMotionProperties();
             if (motionProperties)
             {
-                return 1.0f / motionProperties->GetInverseMass();
+                const f32 invMass = motionProperties->GetInverseMass();
```

Cppcheck (2.20.0)

[style] The function 'ComposeTransform' is never used.

(unusedFunction)